ioctl/newcmd/alias/aliasexport.go

```diff
@@ -48,33 +48,33 @@ func NewAliasExport(c ioctl.Client) *cobra.Command {
 	ec := &cobra.Command{
 		Use:   use,
 		Short: short,
-		Args:  cobra.ExactArgs(0),
+		Args:  cobra.ExactArgs(1),
```

Comment:
This cmd should be `ioctl alias export -f ...`, this is clear.  so format can be set by Flag().  pls see example
https://github.com/iotexproject/iotex-core/blob/master/ioctl/newcmd/account/accountcreate.go#L100